cfgov/unprocessed/js/modules/beta-banner-state.js

```diff
@@ -5,38 +5,49 @@
 
 'use strict';
 
-// Import modules
-var $ = require( 'jquery' );
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+
+// Required modules.
+var Expandable = require( '../molecules/Expandable' );
 var _storage = require( './util/web-storage-proxy' );
 
 // Private variables.
-var _key = 'betaBannerIsCollapsed';
+var _expandable;
+
+// Constants.
+var COLLAPSED_STATE = 'betaBannerIsCollapsed';
+
 
 /**
  * Set up DOM references and event handlers.
  */
 function init() {
 
-  // DOM references.
-  var btnDOM = $( '#beta-banner_btn' );
-  var bannerDOM = $( '#beta-banner' );
+  // Init Expandable.
+  var betaBannerDom = document.querySelector( '#beta-banner' );
+  var isExpanded = _storage.getItem( COLLAPSED_STATE ) !== 'true';
```

Comment:
Got it. The naming is a bit confusing, maybe we should rename the stored value as `EXPANDED_STATE` to make it clearer?